.github/actions/get-merge-commit/action.yml

```diff
@@ -0,0 +1,35 @@
+name: "Get merge commit"
+
+outputs:
+  mergedSha:
+    description: "The merge commit SHA"
+    value: ${{ steps.merged-sha-generator.outputs.mergedSha }}
+
+runs:
+  using: "composite"
```

Comment:
I am not sure what I am doing wrong here. Tried my best to follow: https://github.com/storacha/add-to-web3/blob/main/action.yml

which gives an output as well.

https://github.com/JohnRTitor/nixpkgs/actions/runs/12164535023/job/33926420934?pr=2 currently does not run at all.

Reply:
CC @azuwis

Reply:
I think it would be better to have this as a seperate workflow, in a seperate repo.

Or we can just revert to the previous solution.

Reply:
@JohnRTitor I believe the error is accurate, the repo needs to be checked out before a local workflow can be used. So this part can't be reused and needs to be duplicated:
```yaml
    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
      with:
        path: base
        sparse-checkout: ci
```

Reply:
Discussed [on Matrix](https://matrix.to/#/!kjdutkOsheZdjqYmqp:nixos.org/$GCaAXJ7qLyJHFdCNOhANg71BCSxs0ytcRNqwIibj2wE?via=nixos.org&via=matrix.org&via=tchncs.de). Main points:
- This then also requires `rm -rf base` again, kind of annoying
- @JohnRTitor had the idea of turning eval and Nixpkgs-vet to a reusable workflow and call it from a bigger/main workflow, to explore in the future
- For now we can just use reusable workflows, it's not too much extra noise and we can always change it